src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -143,6 +143,8 @@ void* cSTIR_newObject(const char* name)
 			return NEW_OBJECT_HANDLE(RayTracingMatrix);
 		if (sirf::iequals(name, "SPECTUBMatrix"))
 			return NEW_OBJECT_HANDLE(SPECTUBMatrix);
+		if (sirf::iequals(name, "PinholeSPECTUBMatrix"))
```

Comment:
needs `#if STIR_VERSION >= 050100` otherwise `PinholeSPECTUBMatrix` is not defined